src/elli_request.erl

```diff
@@ -53,8 +54,10 @@
 path(#req{path = Path})          -> Path.
 %% @doc Return the `raw_path', i.e. not split or parsed for query params.
 raw_path(#req{raw_path = Path})  -> Path.
-%% @doc Return the `headers'.
+%% @doc Return the original `headers'.
 headers(#req{headers = Headers}) -> Headers.
+%% @doc Return the `headers' that have had `string:casefold/1' run on each key.
```

Comment:
should these two docstrings be flipped?

Reply:
Hehe, yea, got screwed up when I switched to using `original_headers` :)